data/remnant/remnant outfits.txt

```diff
@@ -381,6 +381,26 @@ outfit "Emergency Ramscoop"
 	"ramscoop" 0.5
 	description "The Ember Waste is the sort of place where only a very foolish captain would travel without some sort of device for replenishing hyperspace fuel. While this risk is non-existent in Remnant ships due to their 'gills', which are effectively built-in ramscoops, some captains prefer to speed up the process by adding additional capacity. For the rare captain braving the area in non-Remnant ships, this ramscoop is not particularly powerful, but it allows a ship to explore without fear of getting stranded."
 
+outfit "Research Laboratory"
+	category "Systems"
+	licenses
+		Remnant
+	cost 500000
+	thumbnail "outfit/remnant laboratory"
+	"mass" 11
+	"outfit space" -6
+	"asteroid scan power" 60
+	"cargo scan power" 10
+	"cargo scan speed" 1
+	"outfit scan power" 10
+	"outfit scan speed" 1
+	"tactical scan power" 10
+	"atmosphere scan" 100
+	"required crew" 1
+	"unplunderable" 1
+	description "From the very beginning Remnant have relied on their ability to study everything they encounter. This knowledge has enabled them to survive and even prosper in the harsh environment of the Ember Waste. To enable this pursuit of knowledge, the Remnant designed this compact laboratory that is built onto the side of a cargo bay, mess hall, or other larger room. When the room isn't otherwise occupied, this module expands out to convert it into a full fledged laboratory."
```

Comment:
```suggestion
	description "From the very beginning, the Remnant have relied on their ability to study everything they encounter. This knowledge has enabled them to survive and even prosper in the harsh environment of the Ember Waste. To enable this pursuit of knowledge, the Remnant designed this compact laboratory that is built onto the side of a cargo bay, mess hall, or other larger room. When the room isn't otherwise occupied, this module expands out to convert it into a full fledged laboratory."
```